controllers/ramenconfig.go

```diff
@@ -221,12 +223,17 @@ func ConfigMapGet(
 	ctx context.Context,
 	apiReader client.Reader,
 ) (configMap *corev1.ConfigMap, ramenConfig *ramendrv1alpha1.RamenConfig, err error) {
+	configMapName := HubOperatorConfigMapName
+	if ControllerType != ramendrv1alpha1.DRHub {
+		configMapName = drClusterOperatorConfigMapName
+	}
+
```

Comment:
(nit) this if could be removed by reusing the one on in main.go line 88

Reply:
I am not sure how the `if` can be removed. We are using the global variable `ControllerType` set in main.go